src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -143,6 +135,22 @@ trait SharedFileSystem {
     }
   }
 
+  private def postProcessInner(backendCall: LocalFileSystemBackendCall, taskOutputs: Seq[TaskOutput], listBeingBuilt: Seq[(String, Try[WdlValue])]): Seq[(String, Try[WdlValue])] = {
+    taskOutputs match {
+      case Nil => listBeingBuilt
+      case head :: tail =>
+        val taskOutput = head
+        val alreadyEvaluatedExpressions: Map[String, WdlValue] = listBeingBuilt filter { _._2.isSuccess } map { case(x,y) => (x,y.get) } toMap
+        val tryConvertedValue =
+          for {
+            expressionValue <- taskOutput.expression.evaluate(backendCall.lookupFunction(alreadyEvaluatedExpressions), backendCall.engineFunctions)
+            convertedValue <- outputAutoConversion(backendCall, taskOutput, expressionValue)
+            pathAdjustedValue <- Success(absolutizeOutputWdlFile(convertedValue, backendCall.callRootPath))
+          } yield pathAdjustedValue
+        postProcessInner(backendCall, tail, listBeingBuilt ++ Seq((taskOutput.name, tryConvertedValue)))
+    }
+  }
+
```

Comment:
There seems to be a lot of duplication between this and what's in `JesBackend`.  Can the core of the code be centralized and only the differences factored out?